targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F412xG/system_clock.c

```diff
@@ -167,7 +167,7 @@ uint8_t SetSysClock_PLL_HSE(uint8_t bypass)
 /******************************************************************************/
 /*            PLL (clocked by HSI) used as System clock source                */
 /******************************************************************************/
-uint8_t SetSysClock_PLL_HSI(void)
+MBED_WEAK uint8_t SetSysClock_PLL_HSI(void)
```

Comment:
For my information, I agree that WEAK to SetSysClock or to SetSysClock_PLL_HSE
could be useful, but for this SetSysClock_PLL_HSI, what is the issue ?
This should work as HSI input clock is the same ?

Reply:
You are right, the board configuration should not affect the HSI. Indeed, it works using the default code for this part. In this case, we can remove this weak statement.